x/ibc/04-channel/client/cli/query.go

```diff
@@ -67,15 +70,15 @@ func GetCmdQueryChannel(clientCtx client.Context) *cobra.Command {
 		),
 		Args: cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx = clientCtx.Init()
-
-			portID := args[0]
-			channelID := args[1]
-			prove, err := cmd.Flags().GetBool(flags.FlagProve)
+			clientCtx, err := client.ReadQueryCommandFlags(clientCtx, cmd.Flags())
 			if err != nil {
 				return err
 			}
 
+			portID := args[0]
+			channelID := args[1]
```

Comment:
do you think we should add naming validation checks to these args? this could be a followup since there might be other args we'd want to check

Reply:
the gRPC request validates the identifiers. Is that what you mean?

Reply:
ah yea that works and makes sense. Just wanted to double check you couldn't query for a portID that couldn't possibly exist